go.mod

```
@@ -1,6 +1,8 @@
 module go.k6.io/k6
 
-go 1.20
+go 1.21
+
+toolchain go1.21.11
```

```suggestion
toolchain go1.21.13
```

There is a later version ;)